### PR TITLE
服务器资源cmdb信息展示添加分类配置

### PR DESCRIPTION
**Pull request by leowucn**

### CODE REVIEW

**Comment by openc3-bot-001** [comment]
test/ok

build-c3-front:ok
check-yml-file-format:ok
compiler-bl-proxy:ok
compiler-jumpserver:ok
compiler-perl:ok
compiler-trouble-ticketing:ok

File: AGENT/device/conf/template/sync-google-vm.yml [hunk 1]
```diff
@@ -43,7 +43,29 @@ download:
         - name: _ingestion_
           alias: 摄取
           select: ["YES", "NO"]
-    check:
```
Comment (lijinfeng2011) [inline AGENT/device/conf/template/sync-google-vm.yml:46]: 此处为误删，请修复。

Comment (lijinfeng2011) [reply AGENT/device/conf/template/sync-google-vm.yml:46]: @leowucn  修复一下

**Comment by openc3-bot-001** [comment]
test/ok

build-c3-front:ok
check-yml-file-format:ok
compiler-bl-proxy:ok
compiler-jumpserver:ok
compiler-perl:ok
compiler-trouble-ticketing:ok

**Pull request merged**